### PR TITLE
Breakout Conversation Functions to Public

### PR DESCRIPTION
**Pull request by Ichbinjoe**
Break out setPrefix to public usage, as well as addConversationCanceller, and added a new function, removeConversationCanceller.

### CODE REVIEW

**Comment by turt2live** [comment]
This PR needs to follow the [CONTRIBUTING.md Guidelines](https://github.com/Bukkit/Bukkit/blob/master/CONTRIBUTING.md) as well as resolve other formatting concerns to be further reviewed.

File: src/main/java/org/bukkit/conversations/Conversation.java [hunk 1]
```diff
@@ -152,11 +152,23 @@ void setPrefix(ConversationPrefix prefix) {
...
-    void addConversationCanceller(ConversationCanceller canceller) {
+    public void addConversationCanceller(ConversationCanceller canceller) {
         canceller.setConversation(this);
         this.cancellers.add(canceller);
     }
 
+
+    /**
+     * Removes a {@link ConversationCanceller} to the cancellers collection.
+     *
```
Comment (gmcferrin) [inline src/main/java/org/bukkit/conversations/Conversation.java:163]: Extra whitespace here needs removing.

File: src/main/java/org/bukkit/conversations/Conversation.java [hunk 2]
```diff
@@ -152,11 +152,23 @@ void setPrefix(ConversationPrefix prefix) {
...
     }
 
+
+    /**
+     * Removes a {@link ConversationCanceller} to the cancellers collection.
+     * 
+     * @param canceller The {@link ConversationCanceller} to remove.
+     */
+    public void removeConversationCanceller(ConversationCanceller canceller){
+        if(canceller.getConversation() == this && this.cancellers.contains(canceller)){
```
Comment (gmcferrin) [inline src/main/java/org/bukkit/conversations/Conversation.java:167]: Spaces after if, and between ) & { as well.

File: src/main/java/org/bukkit/conversations/Conversation.java [hunk 3]
```diff
@@ -152,11 +152,23 @@ void setPrefix(ConversationPrefix prefix) {
...
         this.cancellers.add(canceller);
     }
 
+
+    /**
+     * Removes a {@link ConversationCanceller} to the cancellers collection.
+     * 
+     * @param canceller The {@link ConversationCanceller} to remove.
+     */
+    public void removeConversationCanceller(ConversationCanceller canceller){
```
Comment (gmcferrin) [inline src/main/java/org/bukkit/conversations/Conversation.java:166]: Space between ) & {

**Comment by JeromSar** [comment]
This PR has been stale for several months. What is the policy when dealing with inactive PRs?

**Comment by turt2live** [comment]
@JeromSar If you'd like to take over this PR, please complete the following list of items. So long as this PR remains open and your comments on this PR addressing the issues are up to par, they will be included.
- Contributing guidelines (all components)
- Testing materials
- Diff comments addressed
- Any other known concerns addressed

Code may be submitted by linking the applicable branch on this PR. It is recommended to pull the original author's branch to your own branch before modifying the code.

**Pull request closed without merging**